src/main/java/org/broadinstitute/hellbender/engine/spark/SparkContextFactory.java

```diff
@@ -26,6 +26,9 @@
     public static final Map<String, String> MANDATORY_PROPERTIES = ImmutableMap.<String,String>builder()
             .put("spark.serializer", KryoSerializer.class.getCanonicalName())
             .put("spark.kryo.registrator", "org.broadinstitute.hellbender.engine.spark.GATKRegistrator")
+            // remap the Hadoop FS implementation for file:// URIs to avoid writing CRC files for local files
+            // note that we don't use Hadoop's RawLocalFileSystem since it doesn't extend LocalFileSystem
+            .put("spark.hadoop.fs.file.impl", "org.broadinstitute.hellbender.engine.spark.NonChecksumLocalFileSystem")
```

Comment:
use `NonChecksumLocalFileSystem.class.getCanonicalName()` rather than hardwiring the name.

same for `GATKRegistrator` above